test/cdk-test-integration/README.md

```diff
@@ -0,0 +1,15 @@
+# Welcome to your CDK TypeScript project
```

Comment:
You should either delete this file, or update the README here to explain that this CDK app exists within your construct library to act as an integration test